serenity-report-resources/src/main/resources/freemarker/menu.ftl

```diff
@@ -40,7 +40,7 @@
         </li>
     </#if>
     </ul>
-    <span class="date-and-time"><a href="build-info.html"><i class="bi bi-info-circle"></i></a> Report generated ${timestamp}</span>
+    <span class="date-and-time"><a href="build-info.html">Detailed build information <i class="bi bi-info-circle"></i></a> <br> Report generated ${timestamp}</span>
```

Comment:
The <br> tags don't seem to have closing tags?

Reply:
Have been updated, In HTML, the <br /> tag is a self-closing tag and does not require a separate closing tag.